packages/components/src/button/style.scss

```diff
@@ -292,6 +292,16 @@
 	}
 }
 
+.button-bare {
+	appearance: none;
+	background: transparent;
+	border: none;
+	font-size: inherit;
+	outline: 0;
+	padding: 0;
+	vertical-align: baseline;
+}
```

Comment:
These overrides are designed to reset the user agent styles for `<button>`, but `<Button href>` renders a `<a href>` link. I don't see anything wrong right now, but at some moment we might need two distinct CSS classes for `button.button-bare` and `a.button-bare`.

Note that we still have a global styles for `a href` that give the link a Calypso look and feel. Removing these global styles could be a good opportunity to revisit `<Button bare href>`.